packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -63,6 +75,45 @@ const ProductFullDetail = props => {
         />
     ) : null;
 
+    // Fill a map with field/section -> error.
+    const errors = new Map();
+    if (derivedErrorMessage) {
+        Object.keys(ERROR_MESSAGE_TO_FIELD_MAPPING).forEach(key => {
+            if (derivedErrorMessage.includes(key)) {
+                const target = ERROR_MESSAGE_TO_FIELD_MAPPING[key];
+                const message = ERROR_FIELD_TO_MESSAGE_MAPPING[target];
+                errors.set(target, message);
+            }
+        });
+        if (!errors.size) {
```

Comment:
So this is essentially we have errors but we couldn't map them to any fields, right? So we just show a general form-level error message?

Reply:
yes